ecl2df/parameters.py

```diff
@@ -65,9 +66,20 @@ def load_parameterstxt(filename: Union[str, Path]) -> Dict[str, Any]:
         filename: file containing one key-value pair pr. line,
             separated by whitespace
     """
-    dframe = pd.read_csv(
-        filename, comment="#", sep=r"\s", engine="python", names=["KEY", "VALUE"]
-    )
+    with warnings.catch_warnings(record=True):
+        warnings.filterwarnings("error")
```

Comment:
Maybe a small comment here is warranted for the future developer?

Reply:
done